Add suspension banner

iskandersamerkhanov opened the pull request:
👓 @pwildani @schwers @tomsoir 
JIRA Ticket: https://reddit.atlassian.net/browse/AE-983?filter=-1

iskandersamerkhanov commented:
![screenshot from 2017-07-17 15-12-11](https://user-images.githubusercontent.com/2392019/28285606-609db28e-6b03-11e7-817b-e339fd766f12.png)

iskandersamerkhanov commented:
![screenshot from 2017-07-17 15-20-16](https://user-images.githubusercontent.com/2392019/28285623-74a90c7e-6b03-11e7-92e3-b9e19984bda2.png)

schwers commented on src/app/components/SuspensionBanner/styles.less:
```diff
@@ -0,0 +1,25 @@
+    margin-left: 60px;
+    padding: 10px;
+    color: white;
+    background-color: #eb758b;
```
🔕  Might be able to use one of the nsfw salmon or error red's here. Non-blocker

kjoconnor commented:
`susspension` -> `suspension`

iskandersamerkhanov commented:
Fixed typo in commit name
Changed colors to '@red' and '@nsfw-salmon'

@schwers Could you please deploy this commit, cause i have not write access?

<img width="374" alt="screen shot 2017-08-07 at 11 56 54 am" src="https://user-images.githubusercontent.com/2392019/29041403-9135fbf8-7b67-11e7-9ee9-ff3026a8af4d.png">

schwers commented:
merged via https://github.com/reddit/reddit-mobile/commit/076b426e9f29210c7d72fcf695cdc03b9e9a915c
